### ISSUE TITLE
Improvement: Is it possible to get it working with domains (username/passwords)?

### ISSUE THREAD

**Issue by MaikNog** [jpkrause/AndroidProxySetter#6]
My typical use case scenario is working in a QA department, where I test certain features with a proxy.
We are in the company wireless network, which is secured via username/password, which is registered in their local LDAP.

The access(?) method used is PEAP / MSCHAPV2.

**Comment by manadream** [comment]
I'll look into this. When I initially created this I didn't see that option readily available in the APL library to implement but I'll take a second look.

**Comment by jireton** [comment]
Thanks for looking at this.  My use case is basically identical to the reported issue PEAP with user/pass

**Comment by manadream** [comment]
Yeah, it looks like this functionality will not work at the moment. The APL does not support EAP security when setting the proxy settings on a WiFi AP. See shouldit/android-proxy#5
That said, I'll leave this open and if I have time I'll try to add this functionality

**Comment by zcrself** [comment]
great, thanks for your reply
